### PR TITLE
Add template styles

### PR DESCRIPTION
**Pull request by vegeorgi**
- Add css file to .gitignore
 - Add page background icons files
 - Add nativescript-dev-sass and node-sass to package.json

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@ npm-debug.log
 
 *.js.map
 **/*.js
+**/*.css
```
Comment (MIYankov) [inline .gitignore]: I believe you should also delete the existing .css files in the repo.

File: shared/my-drawer/MyDrawer.ts [hunk 2]
```diff
@@ -1,29 +1,39 @@
 import { EventData } from "data/observable";
 import { StackLayout } from "ui/layouts/stack-layout";
 
+import view = require("ui/core/view");
```
Comment (MIYankov) [inline shared/my-drawer/MyDrawer.ts]: Per guidelines - system imports should be separated from the app imports by a single line. This is a system import. It should be with the top ones. Also, they should be alphabetically sorted by package. I suggest you install and configure tslint.

File: shared/my-drawer/MyDrawer.ts [hunk 3]
```diff
@@ -1,29 +1,39 @@
...
 import frameModule = require("ui/frame");
 
+class SelectedPageView extends view.View {
```
Comment (MIYankov) [inline shared/my-drawer/MyDrawer.ts]: Per guidelines - models should be in separate files. Also, there is a view-model for the drawer now, check it out.

File: shared/my-drawer/MyDrawer.ts [hunk 4]
```diff
@@ -36,6 +46,18 @@ export function onLoaded(args: EventData) {
     https://docs.nativescript.org/api-reference/classes/_ui_page_.page.html
     */
     const page = <StackLayout> args.object;
+
+    let thisView: SelectedPageView = <SelectedPageView>args.object;
+    if (thisView.selectedPage) {
+        for (var item = 0; item <= model.navigationItems.length - 1; item++) {
+            if (model.navigationItems[item].title === thisView.selectedPage) {
+                model.navigationItems[item]["className"] = "sidedrawer-list-item selected";
```
Comment (MIYankov) [inline shared/my-drawer/MyDrawer.ts]: Can't we bind this to a function that returns the class name?

**Comment by vegeorgi** [comment]
Make a new one

**Pull request closed without merging**